OpenAIAssistants/Source/OpenAI+Runs.swift

```diff
@@ -39,6 +39,50 @@ public extension OpenAI {
             .eraseToAnyPublisher()
     }
 
+    func createStreamRun(in threadId: String, payload: CreateRunPayload) -> AnyPublisher<Message, StreamError> {
+        let subject = PassthroughSubject<Message, StreamError>()
+        let url = URL(string: "https://api.openai.com/v1/threads/\(threadId)/runs")!
```

Comment:
shouldn't the "baseUrl" be used here?

Reply:
also this can potentially produce a broken url depending on what comes in "threadId". I would add more sanity checks

Reply:
Hi, can you please see and merge a https://github.com/exyte/openai-assistants-api/pull/4

the last v2 changes (common baseURL) located in this pr